src/routes/api/home/data.js

```diff
@@ -50,6 +50,123 @@ export const hot_deals = [
   }
 ]
 
+export const gift_cards = [
+    {
+      name: 'Amazon Gift Card US',
+      product_type: 'Gift Cards',
+      url: '/store/gift-cards/amazon-card',
+      imgUrl: 'https://shopvian.com/wp-content/uploads/2021/07/778f2cee60f050175dbbee93d205e194.png',
+      country: 'US',
+      brand: 'Amazon',
+      categories: ['shopping', 'gift cards'],
+      price_tags: {
+        min: 25,
+        denominations: [25, 50, 100, 200],
+        flexible: true,
+        commission: 3,
+      },
+      discount: 5,
+      discount_until: 7,
+      faq: `
+        <h3>Redemption Instructions:</h3>
+        <p>1. Go to <a href="https://www.amazon.com/redeem" target="_blank">amazon.com/redeem</a></p>
+        <p>2. Enter your code and click apply</p>
+        <p>Your balance will be updated immediately.</p>
+      `,
+    },
```

Comment:
I think you need to streamline this data as well to what you need for that page.

If I get what you were trying to do, which is to define what keys are needed overall for the games data (like in the description page), then what you should do is to define a separate games_data export that contains the full spec of the games data. You can then use that games_data for the description page, while keeping this list light for just the games listing page